crates/driver/src/infra/notify/liquidity_sources/mod.rs

```diff
@@ -0,0 +1,40 @@
+pub mod config;
```

Comment:
I think this module definitely needs a good explainer as it's not obvious why it's important to announce somewhere what liquidity sources you are using.